### PR TITLE
runfix: correct missing paragraph tags [ACC-321]

### PR DESCRIPTION
**Pull request by V-Gira**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-321" title="ACC-321" target="_blank">SubtaskACC-321</a>  Open issues 9.1.3.1d structured content
  </td></table>
  <br />
 

### Issue
- Addesso pointed out missing paragraph tags in their linked review

### Fixes
- address the Addesso review
- fix a modal overflow issue with large fonts

### CODE REVIEW

File: src/script/page/AppLock/AppLock.tsx [hunk 1]
```diff
@@ -292,14 +292,14 @@ const AppLock: React.FC<AppLockProps> = ({
...
         {state === APPLOCK_STATE.SETUP && (
           <form onSubmit={onSetCode}>
-            <div
+            <p
               className="modal__text"
               dangerouslySetInnerHTML={{__html: t('modalAppLockSetupMessage', {}, {br: '<br><br>'})}}
               data-uie-name="label-applock-set-text"
             />
-            <div className="modal__text modal__label" data-uie-name="label-applock-unlock-text">
+            <label className="modal__text modal__label" data-uie-name="label-applock-unlock-text">
```
Comment (przemvs) [inline src/script/page/AppLock/AppLock.tsx]: Hmm.. There should be a htmlFor value? 🤔

File: src/script/page/MainContent/panels/preferences/devices/DevicesPreferences.tsx [hunk 2]
```diff
@@ -91,13 +91,15 @@ const Device: React.FC<{
             {device.getName()}
           </div>
 
-          <div className="preferences-devices-id">
-            <span>{t('preferencesDevicesId')}</span>
+          <p className="preferences-devices-id">
+            <span>
+              <strong>{t('preferencesDevicesId')}</strong>
+            </span>
```
Comment (przemvs) [inline src/script/page/MainContent/panels/preferences/devices/DevicesPreferences.tsx]: We need this additional tag?

Comment (przemvs) [inline src/script/page/MainContent/panels/preferences/devices/DevicesPreferences.tsx]:
```suggestion
          <strong>{t('preferencesDevicesId')}</strong>
```

File: src/script/page/MainContent/panels/preferences/devices/components/DetailedDevice.tsx [hunk 3]
```diff
@@ -39,7 +39,9 @@ const DetailedDevice: React.FC<DeviceProps> = ({device, fingerprint}) => {
       </h3>
 
       <p className="preferences-devices-id">
-        <span>{t('preferencesDevicesId')}</span>
+        <span>
+          <strong>{t('preferencesDevicesId')}</strong>
+        </span>
```
Comment (przemvs) [inline src/script/page/MainContent/panels/preferences/devices/components/DetailedDevice.tsx]: We need this 2 tags ?
```suggestion
      <strong>{t('preferencesDevicesId')}</strong>
```

**Review by PatrykBuniX — APPROVED** [review]

**Pull request merged**